### PR TITLE
testing branch

### PR DESCRIPTION
**Pull request by jvendryes18**
Added includes for chapters 1-12, and this is a test to show you what an MR/Pull Request looks like.

### CODE REVIEW

**Comment by coulston** [comment]
Thanks for performing this experiment.  I learned a lot, but still made some mistakes.  For example, I merged test_branch before I looked at this pull request.  I reviewed the changed in GitHub desktop, but could not find out how to leave you feedback.  I think that I could have done that by looking at the Pull Requests tab on the repositories GitHub page.  Appreciate your patience as I get this workflow figured out :)

**Comment by jvendryes18** [comment]
Yeah usually in the pull requests link it lets you make comments on the main page, or you can even go into the 'files changed' tab and leave comments for specific lines of code as well

File: Latex/hwSolutions.tex [hunk 1]
```diff
@@ -60,5 +60,16 @@
...
 \include{./chapter01/problemSet01}
+\include{./chapter02/problemSet02}
+\include{./chapter03/problemSet03}
+\include{./chapter04/problemSet04}
+\include{./chapter05/problemSet05}
+\include{./chapter06/problemSet06}
+\include{./chapter07/problemSet07}
+\include{./chapter08/problemSet08}
+\include{./chapter09/problemSet09}
+\include{./chapter10/problemSet10}
```
Comment (jvendryes18) [inline Latex/hwSolutions.tex:71]: like this (if you ever see this comment)

**Comment by jvendryes18** [comment]
My work also has a separate 'release' branch where they merge all other changes into

for example:
main
release
feature

main - your main
release - copy of main that is still in testing
feature - other branches to be merged into release

then, only once you are sure your release branch is finalized, you can merge that release branch into main, and make a new release branch for the next release, while letting others use your main branch

File: Latex/hwSolutions.tex (hunk 1 above)
Comment (coulston) [reply Latex/hwSolutions.tex:71]: # Realization
You can continue working on a branch even after its been merged.  Hmmm...
And everything is markdown :bowtie:

**Comment by coulston** [comment]
#5 
Oh wait, maybe you can't continue to work on a branch after it's been merged 🫤

**Comment by coulston** [comment]
You just make a never-ending series of release branches.  Once one is merged, create another.💡

**Pull request merged**